Client/src/components/Pages/Feedback/Feedback.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+import Button from '../../layout/Buttons/Button';
+import FormElement from '../../layout/Form/FormElement';
+import Title from '../../layout/Typography/Title/Title';
+
+function Feedback() {
+  const [email, setemail] = useState('');
+  const [message, setmessage] = useState('');
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    const user = {
+      username: email,
+      message: message
+    };
+    console.log('SUBMIT BABY', user);
+    localStorage.setItem('feedback', JSON.stringify(user));
+    setemail('');
+    setmessage('');
+  };
+
+  return (
+    <div>
+      <Title>Feedback</Title>
+      <p>
+        Dear User, your feedback is important for us. Please feel free to let us know what you think
+        about this app.
+      </p>
+      <form className="form" id="form">
+        <FormElement>
+          <input
+            type="email"
+            name="email"
+            value={email}
+            placeholder="Email..."
+            onChange={event => setemail(event.target.value)}
+          />
+        </FormElement>
+        <textarea
+          placeholder="Message..."
+          rows="15"
+          id="text"
+          name="message"
+          value={message}
+          onChange={event => setmessage(event.target.value)}
+        ></textarea>
+        <Button name="Send" click={event => handleSubmit(event)} />
```

Comment:
1) I suggest to use similar property names as you would use with normal buttons... such as onClick in your case. 
2) Not sure if you really need to preventDefault in this case. (your button is not of type submit)

Reply:
Good point. Thank you. I had to specify it as a type button in order to omit the prevent default.